ftw/addressblock/geo/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records interface="collective.geo.settings.interfaces.IGeoFeatureStyle">
+        <value key="map_viewlet_position">plone.belowcontentbody</value>
+    </records>
+
+    <records prefix="plone.resources/ftw-addressblock-js-authenticated"
+             interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+        <value key="expression">python: member is not None</value>
```

Comment:
Have you tested this? 
The `expression` is a field on the bundle not on the js itself.
https://github.com/plone/Products.CMFPlone/blob/9ae305beb3abd55cf47e05425e19f201768c804a/Products/CMFPlone/interfaces/resources.py#L63-L68

Reply:
No, I did not. I was relying on your comment how to do it in issue #26.
Also I would expect the tests to fail if is relevant to the security.

Reply:
Sorry, my issue should have been more clear.

Reply:
I think I fixed this now.